Shoko.Server/Providers/AniDB/UDP/Info/RequestGetFile.cs

```diff
@@ -222,10 +222,10 @@ protected override UDPResponse<ResponseGetFile> ParseResponse(UDPResponse<string
                     }
 
                     // audio languages
-                    var alangs = parts[10].Split(new[] { '\'' }, StringSplitOptions.RemoveEmptyEntries).ToList();
+                    var alangs = parts[10].Split(new[] { '\'' }, StringSplitOptions.RemoveEmptyEntries).Where(lang => lang != "none").ToList();
```

Comment:
Did you confirm that it may include "none" as a value in the list? Just asking to confirm.